src/external/json/json.hpp

```diff
@@ -14431,7 +14431,7 @@ class basic_json
         {
             basic_json&& key = init.begin()->moved_or_copied();
             push_back(typename object_t::value_type(
-                          std::move(key.get_ref<string_t&>()), (init.begin() + 1)->moved_or_copied()));
+                          std::move(key.template get_ref<string_t&>()), (init.begin() + 1)->moved_or_copied()));
```

Comment:
Without this, the Cray compiler gave the error
```
 The indicated type name is not allowed.
                       std::move(key.get_ref<string_t&>()), (init.begin() + 1)->moved_or_copied()));
                                             ^

```

Reply:
I'll take this change, but I will not guarantee that it won't break again in the future, as Cray for C++ compilation is not a target for us. I'll probably add it to our CI once #207 is resolved in some way, but in case it fails with a different error on an updated json.hpp I won't spend a lot of time in trying to fix it.